test/logic.js

```diff
@@ -222,7 +222,23 @@ test('detects instagram in-app browser', function (t) {
   t.end();
 });
 
+test('detects crawlers', function (t) {
```

Comment:
maybe rename these tests to `detects crawler: <crawler name>`.  Just having a couple of tests is fine too - definitely doesn't need to be exhaustive.